File: Iniciante/URI 2234.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val H: Double = leitor.nextDouble();
    val P: Double = leitor.nextDouble();
    
    val media: Double = H / P;
    println(String.format("%.2f", media));
}

File: Iniciante/URI 2234.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val H = leitor.nextDouble()
        val P = leitor.nextDouble()
        val media = H / P
        printf("%.2f\n", media)
    }
	
}